gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -231,9 +235,23 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
       mapContainerFragment = restoreChildFragment(savedInstanceState, MapContainerFragment.class);
     }
 
+    updateNavHeader();
     setUpBottomSheetBehavior();
   }
 
+  private void updateNavHeader() {
+    View navHeader = binding.navView.getHeaderView(0);
+    User user = authenticationManager.getCurrentUser();
+
+    TextView userDisplayNameTv = navHeader.findViewById(R.id.user_display_name);
```

Comment:
Can data binding + VM be used in the layout  instead of `findViewById`?

Reply:
Done.